events/2019/11-contributor-summit/README.md

```diff
@@ -0,0 +1 @@
+# 2018 Kubernetes Contributor Summit NA
```

Comment:
This should have been 2019? :stuck_out_tongue:

Reply:
LOL, yes, fixed here https://github.com/kubernetes/community/pull/3248 😄